packages/terra-navigation-prompt/docs/README.md

```diff
@@ -0,0 +1,138 @@
+# Terra Navigation Prompt
+
+The NavigationPrompt and NavigationPromptCheckpoint components allow for the detection and management of components with unsaved state.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-navigation-prompt`
```

Comment:
Out of curiosity, should an application only pull in one version of navigation prompt, like react?

 If so, I think it would be helpful to expand on how this should be pulled into a project as a dev-dependency and peer dependency to later be pulled into an different project which is the application.

Reply:
I too think that would help.